src/Disks/IO/ReadBufferFromAzureBlobStorage.cpp

```diff
@@ -268,12 +246,10 @@ size_t ReadBufferFromAzureBlobStorage::readBigAt(char * to, size_t n, size_t ran
         size_t bytes_copied = 0;
         try
         {
-            Azure::Storage::Blobs::DownloadBlobOptions download_options;
+            Azure::Storage::Blobs::DownloadBlobToOptions download_options;
             download_options.Range = {static_cast<int64_t>(range_begin), n};
-            auto download_response = blob_client->Download(download_options);
-
-            std::unique_ptr<Azure::Core::IO::BodyStream> body_stream = std::move(download_response.Value.BodyStream);
-            bytes_copied = body_stream->ReadToCount(reinterpret_cast<uint8_t *>(to), body_stream->Length());
+            auto download_response = blob_client->DownloadTo(reinterpret_cast<uint8_t *>(to), n, download_options);
```

Comment:
Does it really give a performance improvement? Seems like just a different api (and this new way with `DownloadTo` is better and more clear, so the change is good) but it seems to be not to be improving the perf anyhow.